script/weblatedl.go

```diff
@@ -0,0 +1,180 @@
+// Copyright (C) 2022 The Syncthing Authors.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//go:build ignore
+// +build ignore
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"regexp"
+	"sort"
+	"strings"
+)
+
+type stat struct {
+	Code       string `json:"code"`
+	Name       string `json:"name"`
+	Total      int    `json:"total"`
+	Translated int    `json:"translated"`
+	Fuzzy      int    `json:"fuzzy"`
+}
+
+type translation map[string]string
+
+func main() {
+	log.SetFlags(log.Lshortfile)
+
+	if t := authToken(); t == "" {
```

Comment:
Completely nitty and optional: I'd just get the token from env here and keep the variable around, and then pass it to `req` - no function needed.

Reply:
Done.